CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 ### Added
+- Add support for icon theme from https://github.com/Peltoche/lsd/pull/707 [kWeiZh](https://gitHub.com/zwpaper)
```

Comment:
`[kWeiZh](https://gitHub.com/zwpaper)` : Typo `kWeiZh` ?

We can probably also remove the PR number for consistency.

Reply:
that's my other username, let me change it back to the origin one, drop the PR number and do the rebase work.

Reply:
happened to find a typo, `gitHub` -> `github` https://github.com/Peltoche/lsd/blob/master/CHANGELOG.md?plain=1#L23

I will also change it for this minor change🤣

Reply:
Woops, that was my Emacs being helpful :D. Thanks for that.

Reply:
not intelligent enough to also change the capital `G` for `GitHub`😁